scripts/rotate-auth-secrets.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -e
+
+[[ -z "$COPS_HTACCESS_FILE" ]] && echo "Error: COPS_HTACCESS_FILE not set" && exit 1
+[[ -s "$COPS_HTACCESS_FILE" ]] || ( echo "Error: COPS_HTACCESS_FILE does not exist or is empty" && exit 1 )
+
+docker secret create basic-auth-users-temp "$COPS_HTACCESS_FILE"
```

Comment:
Can this used `auth_secret_name_temp` similar to the use of `auth_secret_name` in L49